source/css/sass/mixins/_typography.scss

```diff
@@ -4,10 +4,13 @@
 @import "../variables/font";
 @import "spacing";
 @import "utilities";
+@import "scale";
 
 @mixin set-font-size-and-line-height($font-size, $block-size: $baselinegrid-space-small) {
-  @include rem(font-size, $font-size);
-  line-height: $block-size / $font-size;
+  @include rem((
+    font-size: $font-size,
+    line-height: $block-size,
```

Comment:
This means the line height is now set with rem rather than a unitless value. Avoids trickery with having to cater for rounding differences, and _should_ be fine?

Reply:
Would prefer to keep `line-height` a unitless number: it's best practice and not using it could be considered a code smell. When using a length value, if the local font size changes, the line height won't scale with it and so will have to be independently set again too. Would probably work as long as all line heights are set using this mixin, but it seems brittle.

Reply:
90ca7bbc98bf6629a0c996c05d9f01429b2af4a5